changelog

```diff
@@ -1,3 +1,477 @@
+Version 0.10.0
+  * Breaking changes:
+    * The accompany objective on mission npcs no longer implicitly sets the save objective. Plugin missions that had "accompany" but not "save" should be changed to include "save" if the intention of the mission is that the npcs live (without the bug this time). (@Amazinite, @RisingLeaf)
+  * Big changes:
+    # Content
+    * Hai Reveal! (@MasterOfGrey)
+      * Various fixes to missions. (@bene-dictator, @Hurleveur, @McloughlinGuy, @RisingLeaf, @UnorderedSigh)
+      * Various typo fixes. (@bene-dictator, @EjoThims, @Hatrask, @McloughlinGuy, @tibetiroka, @UnorderedSigh, @williaji)
+      * Relocate the new "Deep Space" systems. (@Quantumshark)
+      * Various fixes to other content. (@TomGoodIdea)
+    * A new alien faction: The Gegno. (@Saugia)
+      * TODO: what is included here.
+      * Improve some wording. (@Zitchas)
+    # UI
+    * The job board supports sorting jobs! (@alextd, @samrocketman)
+    * Text and choices in conversations can be hidden with conditions! (@SolraBizna, @UnorderedSigh, @Hurleveur)
+    * It is now possible to aim your ship and fire primary weapons using your mouse! (@CAPTAIN1947, @samrocketman, @Hurleveur, @Terin)
```

Comment:
We also have a change that allows the ammo icons to be clickable. Perhaps move this line down to the UI section, then have a general "big changes" line that describes how the mouse alone can now be used for more things.